<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>LinearMotorPropDialog</class>
 <widget class="QDialog" name="LinearMotorPropDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Linear Motor Properties</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox3" >
     <property name="title" >
      <string>Size and Color</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="widthLineEdit" >
        <property name="alignment" >
         <set>Qt::AlignLeading</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="textLabel3" >
        <property name="text" >
         <string>Angstroms</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="sradiusLineEdit" >
        <property name="alignment" >
         <set>Qt::AlignLeading</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="lengthLineEdit" >
        <property name="alignment" >
         <set>Qt::AlignLeading</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="textLabel3_3" >
        <property name="text" >
         <string>Angstroms</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="textLabel3_2" >
        <property name="text" >
         <string>Angstroms</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="jig_color_pixmap" >
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="scaledContents" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="choose_color_btn" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Choose...</string>
            </property>
            <property name="autoDefault" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>46</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel1_3" >
        <property name="text" >
         <string>Motor Length:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel1_2_2" >
        <property name="text" >
         <string>Motor Width:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="textLabel1_2_2_2" >
        <property name="text" >
         <string>Spoke Radius:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="colorTextLabel" >
        <property name="text" >
         <string>Color:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_btn" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="nameTextLabel" >
         <property name="text" >
          <string>Name:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1" >
         <property name="font" >
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text" >
          <string>Force:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>lineEdit9</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_2" >
         <property name="text" >
          <string>Stiffness:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>lineEdit9_2</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QLineEdit" name="forceLineEdit" >
         <property name="toolTip" >
          <string>Simulations will begin with the motor's force set to this value.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLabel" name="textLabel2" >
         <property name="text" >
          <string>N/m</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLabel" name="textLabel1_4" >
         <property name="text" >
          <string>pN</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2" >
        <widget class="QLineEdit" name="nameLineEdit" >
         <property name="toolTip" >
          <string>Name of Linear Motor</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading</set>
         </property>
         <property name="readOnly" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLineEdit" name="stiffnessLineEdit" >
         <property name="toolTip" >
          <string>Simulations will begin with the motor's stiffness set to this value.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_5" >
       <property name="toolTip" >
        <string>If checked, the force value is applied to the motor's atoms during minimization.</string>
       </property>
       <property name="text" >
        <string>Enable in Minimize (experimental) :</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enable_minimize_checkbox" >
       <property name="toolTip" >
        <string>If checked, the force value is applied to the motor's atoms during minimization.</string>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>92</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>forceLineEdit</tabstop>
  <tabstop>stiffnessLineEdit</tabstop>
  <tabstop>enable_minimize_checkbox</tabstop>
  <tabstop>lengthLineEdit</tabstop>
  <tabstop>widthLineEdit</tabstop>
  <tabstop>sradiusLineEdit</tabstop>
  <tabstop>choose_color_btn</tabstop>
  <tabstop>ok_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>LinearMotorPropDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok_btn</sender>
   <signal>clicked()</signal>
   <receiver>LinearMotorPropDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
